thoth/advise_reporter/advise_reporter.py

```diff
@@ -63,24 +50,13 @@ def retrieve_adviser_reports_justifications(adviser_version: str):
     return advise_justifications
 
 
-def send_metrics_to_pushgateway(advise_justification: MessageBase):
-    """Send metrics to Pushgateway."""
-    _METRIC_ADVISE_TYPE.labels(advise_message=advise_justification.message, thoth_environment=_THOTH_ENVIRONMENT).set(
-        advise_justification.count
-    )
-    _LOGGER.info("advise_message_number(%r)=%r", advise_justification.message, advise_justification.count)
-
-    if _THOTH_METRICS_PUSHGATEWAY_URL:
-        try:
-            _LOGGER.info(f"Submitting metrics to Prometheus pushgateway {_THOTH_METRICS_PUSHGATEWAY_URL}")
-            push_to_gateway(_THOTH_METRICS_PUSHGATEWAY_URL, job="advise-error-analysis", registry=prometheus_registry)
-        except Exception as e:
-            _LOGGER.info(f"An error occurred pushing the metrics: {str(e)}")
-
-
+@metrics.exceptions.count_exceptions()
+@metrics.in_progress.track_inprogress()
 def expose_metrics(advise_justification: MessageBase):
     """Retrieve adviser reports justifications."""
-    _METRIC_ADVISE_TYPE.labels(advise_message=advise_justification.message, thoth_environment=_THOTH_ENVIRONMENT).set(
-        advise_justification.count
-    )
+    metrics.advise_justification_type.labels(
```

Comment:
I'm confused about what this is exposing

Reply:
It's exposing the number of justifications provided by adviser per type of justification, so we can monitor what we provide and help in deciding what needs to be prioritized